lib/foodcritic/rules.rb

```diff
@@ -26,16 +26,7 @@
   end
 end
 
-rule "FC003",
-     "Check whether you are running with chef server before using"\
-     " server-specific features" do
-  tags %w{portability solo}
-  recipe do |ast, filename|
-    unless checks_for_chef_solo?(ast) || chef_solo_search_supported?(filename)
-      searches(ast)
-    end
-  end
-end
+# FC003 was yanked and the number should not be reused
```

Comment:
Same as above these aren't actually seen by end users so the wording isn't the important part. That's for the PR against foodcritic-site